backbone.js

```diff
@@ -1673,8 +1673,8 @@
       }
       if (!callback) callback = this[name];
       var router = this;
-      Backbone.history.route(route, function(fragment) {
-        var args = router._extractParameters(route, fragment);
+      Backbone.history.route(route, function(fragment, state) {
```

Comment:
it would be extra awesome if we could add a test for this, I know you didn't write the code.

If you're not up for it (no worries) could you make a platform ticket?

Reply:
Ah, of course, I'll do that. My bad I didn't do that off the bat...